Google.PowerShell/Compute/GceCmdlet.cs

```diff
@@ -110,6 +122,51 @@ private void WriteWarnings(Operation op)
             }
         }
 
+        private void WriteProgress(Operation op, string progressMessage)
+        {
+            int activityId = (int)(op.Id % int.MaxValue ?? 0);
```

Comment:
Why not:

    int activityId = opt.Id.GetValueOrDefault() % int.MaxValue;


On Tue, Aug 9, 2016 at 10:58 AM, Jim Przybylinski <notifications@github.com>
wrote:

> In Google.PowerShell/Compute/GceCmdlet.cs
> <https://github.com/GoogleCloudPlatform/google-cloud-powershell/pull/213#discussion_r74111030>
> :
>
> > @@ -110,6 +122,51 @@ private void WriteWarnings(Operation op)
> >              }
> >          }
> >
> > +        private void WriteProgress(Operation op, string progressMessage)
> > +        {
> > +            int activityId = (int)(op.Id % int.MaxValue ?? 0);
>
> op.Id is indeed an int64?, meaning it could be null.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/GoogleCloudPlatform/google-cloud-powershell/pull/213/files/be8a8c708d9bec2ccca4905e841ae3f873603ded#r74111030>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AD19lw5Hzy1nHcmzt-WGRMViVpHgxPygks5qeL_igaJpZM4JfgLC>
> .
>